apps/bajour/pages/a/[slug].tsx

```diff
@@ -46,6 +47,18 @@ export default function ArticleBySlug() {
     }
   })
 
+  const isFDT = data?.article?.tags.includes('frage-des-tages')
+
+  if (isFDT) {
+    return (
+      <WebsiteBuilderProvider ArticleList={RelatedArticleSlider}>
+        <Container>
+          <FrageDesTagesArticle article={data?.article as ApiV1.Article} className="" />
+        </Container>
+      </WebsiteBuilderProvider>
+    )
+  }
+
   return (
     <WebsiteBuilderProvider ArticleList={RelatedArticleSlider}>
```

Comment:
Wouldn't it to make more sense to do:

```suggestion
    <WebsiteBuilderProvider ArticleList={RelatedArticleSlider} Article={isFDT: FrageDesTagesArticle ?? undefined}>
```

Reply:
I tried implementing it this way but it actually didn't work. But it seems that it's going to be changed anyway because Michael wants the Poll + Comments to be part of an article, not the full content of an article like now

Reply:
so that basically means that it will need to be integrated into an article like a normal block